main.rb

```diff
@@ -0,0 +1,148 @@
+require 'date'
+require_relative 'library/classes/book'
+require_relative 'library/classes/person'
+require_relative 'library/classes/teacher'
+require_relative 'library/classes/student'
+require_relative 'library/classes/rental'
+
+# This class represents the main application for managing books, people, and rentals.
+
+# rubocop:disable Metrics
```

Comment:
- [ ] You did a great job shifting your code from `app.rb` to `main.rb`. I know it wasn't easy to move your code and make the adjustments to meet the requirements and make your application run successfully like it was before. But as per the requirement, you should not disable `rubocop` in the project's code snippet. Please remove the comment and resolve the linter issue before submitting for the next review.